operator/cmd/root.go

```diff
@@ -253,9 +302,12 @@ func runOperator(clientset k8sClient.Clientset, shutdowner hive.Shutdowner) {
 	// We only support Operator in HA mode for Kubernetes Versions having support for
 	// LeasesResourceLock.
 	// See docs on capabilities.LeasesResourceLock for more context.
-	if !capabilities.LeasesResourceLock {
+	// TODO REMOVE || true sabotage after done testing
+	if !capabilities.LeasesResourceLock || true {
 		log.Info("Support for coordination.k8s.io/v1 not present, fallback to non HA mode")
-		onOperatorStart(leaderElectionCtx, clientset)
+		if err := llc.Start(leaderElectionCtx); err != nil {
+			log.WithError(err).Fatal("Failed to start leading")
```

Comment:
Latest iteration of hive added `shutdowner.Shutdown(hive.ShutdownWithError(err))` so we can use that now instead of fatal to allow things to stop gracefully (see line 356).